tests/e2e/taqueria-bootstrap.spec.ts

```diff
@@ -0,0 +1,182 @@
+import { spawnSync } from 'child_process';
```

Comment:
Is there a particular reason you're using spawnSync vs the promisified exec method used in other spec files? One reason I ask is because I have noticed that mixing synchronous and asynchronous stuff has lead to some stability issues when running in the pipeline. If possible I want to keep things async in our tests

Reply:
> I have noticed that mixing synchronous and asynchronous stuff has lead to some stability issues when running in the pipeline.

Didn't realize that. 

> If possible I want to keep things async in our tests

Sounds good! All for it.

> Is there a particular reason you're using spawnSync vs the promisified exec method used in other spec files?

I spent some time testing exec, spawn, and execa (an NPM package) in various environments. I found that spawn is more reliable than exec as you can specify `{shell: true}` in the opts.

execa is actually the best of them all, but its an ESM module which our testing framework doesn't seem to support at this time. I did actually get it working in a separate PR. We can chat about this more on Monday if you'd like.